client/src/environments/environment.prod.ts

```diff
@@ -7,4 +7,5 @@ export const environment = {
   production: true,
   api: 'https://api.momentum-mod.org',
   auth: 'https://auth.momentum-mod.org',
+  sentry: { dsn: '', env: 'client-production' },
```

Comment:
Even assuming we're happy with the fact we pass the Sentry DSN to the client, I don't think that should mean we store it publicly on Github where it's more visible and potentially scraped and spammed - I dunno what/who would, but seems like an unnecessary risk. Better to have it loaded from the .env file, it's already defined in there on the NestJS branch as `SENTRY_DSN` in `.env`.